diagnostic.md

```diff
@@ -103,6 +176,7 @@ JSON.
   "name": "Krüller"
 }
 ```
+curl --request PATCH http://www.example.com/
```

Comment:
Not correct - you're missing a bunch of information here. First, we need to specify which donut we're editing.
Second, you need to actually pass in the new data that you're trying to get added to that donut.

And, again, you need `--include`